tests/lib/cli-engine/cli-engine.js

```diff
@@ -923,13 +923,15 @@ describe("CLIEngine", () => {
 
             engine = new CLIEngine({
                 parser: "esprima",
-                useEslintrc: false
+                useEslintrc: false,
+                ignore: false
             });
 
-            const report = engine.executeOnFiles(["lib/cli.js"]);
+            const report = engine.executeOnFiles(["tests/fixtures/simple-valid-project/foo.js"]);
```

Comment:
`esprima` doesn't recognize the object spread syntax I used in `lib/cli.js`, so I had to update this.